src/test/java/CommonScriptTest.java

```diff
@@ -188,9 +201,13 @@ protected File assertJarExists(final File searchDirectory, final String detectVe
         return detectJarFile;
     }
 
-    protected Map<String, String> createEnvironment(final boolean downloadOnly) {
+    protected Map<String, String> createEnvironment(final boolean downloadOnly, final boolean setOldJarDownloadDirVar) {
         final Map<String, String> environment = new HashMap<>();
-        environment.put(EnvironmentVariables.DETECT_JAR_DOWNLOAD_DIR.name(), getOutputDirectory().getAbsolutePath());
+        if (setOldJarDownloadDirVar) {
+            environment.put(EnvironmentVariables.DETECT_JAR_PATH.name(), getOutputDirectory().getAbsolutePath());
+        } else {
+            environment.put(EnvironmentVariables.DETECT_JAR_DOWNLOAD_DIR.name(), getOutputDirectory().getAbsolutePath());
+        }
         environment.put(EnvironmentVariables.DETECT_DOWNLOAD_ONLY.name(), downloadOnly ? "1" : "");
```

Comment:
Getting a little concerned about us having a ton of flags on this method, but this is fine for now.